scheduler/mesosphere-dcos/framework-cleanup.md

```diff
@@ -12,69 +12,69 @@ redirect_from: "/scheduler/mesosphere-dcos/framework_cleanup.html"
 service in this example. These steps can be used to clean up any service in DCOS including the Portworx service.
 
-## 1. Shutdown the service if it is still running
+## Shutdown the service if it is still running
 
 Run `dcos service --inactive` to find the ID of the service that you want to cleanup. The service should in inactive state,
 ie ACTIVE should be set to False.
 
-```
+```bash
 $ dcos service --inactive
-NAME                     HOST            ACTIVE  TASKS  CPU    MEM      DISK   ID                                         
-cassandra-px  ip-10-0-2-15.ec2.internal   False     2    6.7  27530.0  59890.0  cc3a8927-1aec-4a8a-90d6-a9c317f9e8c6-0051  
-marathon              10.0.4.203          True     4    3.1   3200.0    0.0    cc3a8927-1aec-4a8a-90d6-a9c317f9e8c6-0001  
-metronome             10.0.4.203          True     0    0.0    0.0      0.0    cc3a8927-1aec-4a8a-90d6-a9c317f9e8c6-0000  
-portworx      ip-10-0-2-15.ec2.internal   True     6    1.8   5632.0  16384.0  cc3a8927-1aec-4a8a-90d6-a9c317f9e8c6-0050 
+NAME                     HOST            ACTIVE  TASKS  CPU    MEM      DISK   ID
+cassandra-px  ip-10-0-2-15.ec2.internal   False    2    6.7  27530.0  59890.0  cc3a8927-1aec-4a8a-90d6-a9c317f9e8c6-0051
+marathon              10.0.4.203          True     4    3.1   3200.0    0.0    cc3a8927-1aec-4a8a-90d6-a9c317f9e8c6-0001
+metronome             10.0.4.203          True     0    0.0    0.0      0.0    cc3a8927-1aec-4a8a-90d6-a9c317f9e8c6-0000
+portworx      ip-10-0-2-15.ec2.internal   True     6    1.8   5632.0  16384.0  cc3a8927-1aec-4a8a-90d6-a9c317f9e8c6-0050
 ```
 
 Then use the `dcos service shutdown` command to shutdown the service
 
-```
+```bash
 $ dcos service shutdown cc3a8927-1aec-4a8a-90d6-a9c317f9e8c6-0051
 ```
 
-## 2. Run the janitor script to clean up the reserved resources as well as any state stored in Zookeeper
-```
+## Run the janitor script to clean up the reserved resources as well as any state stored in Zookeeper
+```bash
 $ SERVICE_NAME=cassandra-px
 $ dcos node ssh --master-proxy --leader "docker run mesosphere/janitor /janitor.py -r ${SERVICE_NAME}-role -p ${SERVICE_NAME}-principal -z dcos-service-${SERVICE_NAME}"
 ```
 
-## 3. Cleanup PX remnants from slave nodes
+## Cleanup PX remnants from slave nodes
 
 Stop the portworx service on all the nodes and remove the docker container
-```
+```bash
 sudo systemctl stop portworx
 sudo docker rm portworx.service -f
 ```
 
 Remove the portworx service file from all the nodes
-```
+```bash
 sudo rm /etc/systemd/system/portworx.service -f
 sudo rm /etc/systemd/system/dcos.target.wants/portworx.service -f
 sudo systemctl daemon-reload
 ```
-		
-Remove the Portworx config and files from all the nodes
+
+**Note:** If you are going to re-install Portworx, you should wipe out the filesystem from the disks so that they can be picked
```

Comment:
node-wipe was added in 1.3, we still have some customers on 1.2, so we should add a note for that.